README.md

```diff
@@ -1,2 +1,72 @@
-# spark-integration
-Integration tests for Spark
+---
+layout: global
+title: Spark on Kubernetes Integration Tests
+---
+
+# Running the Kubernetes Integration Tests
+
+Note that the integration test framework is currently being heavily revised and
+is subject to change.
+
+Note that currently the integration tests only run with Java 8.
+
+Running the integration tests requires a Spark distribution tarball. It also
+needs a local path to the directory that contains `Dockerimage` files.
```

Comment:
This should say dockerfiles? We're shipping it under `$DISTDIR/kubernetes/dockerfiles` in upstream.